ubg/src/pages/Game.js

```diff
@@ -233,17 +270,33 @@ function Description({usersCollection, game, createRoom, gameId}) {
             </Icon>
             &emsp;
             <AuthCheck>
-              {gameId === '925' ?
-                <Button
-                  variant='contained'
-                  color='primary'
-                  onClick={createRoom}>
-                  Create Room
-                </Button> :
-                null
-              }
-              &emsp;
-              <FavoriteButton usersCollection={usersCollection} game={game}/>
+              <Grid container spacing={3}>
+                {gameId === '925' ?
+                  <Grid item>
+                    <Button
+                      variant='contained'
+                      color='primary'
+                      onClick={createRoom}>
+                      Create Room
+                    </Button>
+                  </Grid> :
+                  null
+                }
+                <Grid item>
+                  <FavoriteButton
+                    usersCollection={usersCollection}
+                    game={game}
+                  />
+                </Grid>
+                <Grid item>
+                  <CreateEventButton
```

Comment:
I think this should only display for mafia, maybe add the same condition as create room?

Reply:
I think they can create event for other games too in case there are other implementations for that game.